src/Backend/Spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from './Command';
+
+/**
+ * There are two type of Spec described on this file.
+ * DeviceSpec : This Spec is a information of cetain Device to specify each Device instance.
+ * BridgeSpec : This Spec will be used on DeviceSpec. this will be used to access on those
+ *              DeviceSpec with Device unique Key.
+ */
+
+class DeviceSpec {
+  hw: string;  // This name comes from 'uname -m'
```

Comment:
Now I agree on using `uname` and `/etc/os-release` but in the future we should find other solutions because the methods could not be there.

Reply:
I add a comment that this `uname` and `/etc/os-release` as a current use, but this could be changed.